### PR TITLE
Ruby group capstone: day 2

### PR DESCRIPTION
**Pull request by nurbolsultanov**
# Hello! This project has been done by @everinurmind and @ghreza-crypto.

**The following requirements have been fulfilled:**
- Each class from the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png) created in a separate .rb file ✔️
- All associations (1-to-many relationships) and aggregations (parent-child relationships) between classes implemented ✔️
- All properties visible in the diagram are defined and set up in the constructor method ✔️
- All methods visible in the diagram are implemented ✔️
- Added unit tests for all implemented methods ✔️
- Created a ``main.rb`` file that will serve as your console app entry-point ✔️
- All data is preserved by saving collections in ``.json`` files ✔️

## [Link](https://drive.google.com/file/d/1x0gkUq6E7lVjyE6K3ocFNEw9z0smPy8x/view) to the Video Description of our Project
Thank you for you attention.

### CODE REVIEW

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good first attempt
✅ Good file structure
✅ Good use of modules to split code and reduce files size

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,138 +1,147 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/everinurmind/Ruby-Group-Capstone-Project/tree/issues).
```
Comment (tufoinnkuo10) [inline README.md:121]: - [optional] please, your issue page link is broken. Kindly add a working one for a professional README file.

File: README.md [hunk 2]
```diff
@@ -1,138 +1,147 @@
...
+
+### Key Features <a name="key-features"></a>
+
+- **Add classes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- WALKTHOROUGH VIDEO -->
+
+## 🎥 Loom Walkthrough <a name="loom-walkthrough"></a>
```
Comment (tufoinnkuo10) [inline README.md:51]: - [ ] Great job adding the video walkthrough link, like the previous reviewer noted. Please, the link is broken. Kindly add a working link so it works well
Screenshot (1145)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TEAM

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @TEAM

Your project is complete! Great job  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- App  works as expected
- Great team work
- No linter errors


video presentation
- Great job making your faces visible throughout the presentation. please always go straight to the point and lay more emphasis on showing how your app works and the different features you were proud of implementing. nice job. 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**